lib/cequel/metal/data_set.rb

```diff
@@ -573,6 +574,12 @@ def page_size(page_size)
         end
       end
 
+      def allow_filtering!
```

Comment:
I think a method comment would be helpful to explain what this is for and which C* version support it.